napari_allencell_annotator/view/annotator_view.py

```diff
@@ -213,12 +218,18 @@ def render_values(self, vals: List):
         vals:List
             the values for the annotations.
         """
-        for (widget, val) in zip(self.annotation_item_widgets, vals):
+        for (widget, val, default) in zip(self.annotation_item_widgets, vals, self.default_vals):
+            if val is None or val == "":
+                val = default
             if isinstance(widget, QLineEdit):
                 widget.setText(val)
             elif isinstance(widget, QSpinBox):
-                widget.setValue(val)
+                widget.setValue(int(val))
             elif isinstance(widget, QCheckBox):
+                if val == "True":
```

Comment:
you can use eval(val) to get the bool here.
If you know the string is going to be `"True"` or `"False"` just use `eval()`